content/en/docs/instrumentation/go/manual.md

```diff
@@ -720,6 +719,76 @@ func init() {
 }
 ```
 
+### Registering Views
+
+A view provides SDK users with the flexibility to customize the metrics output
+by the SDK. You can customize which metric instruments are to be processed or
+ignored. You can also customize aggregation and what attributes you want to
+report on metrics.
+
+Every instrument has a default view, which retains the original name,
+description, and attributes, and has a default aggregation that is based on the
+type of instrument. When a registered view matches an instrument, the default
+view is replaced by the registered view. Additional registered views that match
+the instrument are additive, and result in multiple exported metrics for the
+instrument.
+
+You can use the
+[`NewView`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric#NewView)
+function to create a view and register it using the
+[`WithView`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric#With)
+option.
+
+For example, here's how you create a view that renames the `latency` instrument
+from the `v0.34.0` version of the `http` instrumentation library to
+`request.latency`:
+
+```go
+view := metric.NewView(metric.Instrument{
```

Comment:
Other examples I would include:

* Drop metrics with DropAggregation
* Drop attributes using AttributeFilter

If we add those examples, I would explicitly state that the SDK filters metrics and attributes before recording/storing metrics, allowing Views to be used to reduce memory usage of high cardinality metrics or attributes.

This can be done as a follow-up if needed.

Reply:
It would be probably easier to handle it as a separate review. But if it won't be merged until Monday then I will do my best to address the comment 😉

Reply:
We could also add example how to report as an exponential histogram, how to use wildcards...

I propose to just add a reference that more examples can be found https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric#pkg-examples and we could add more details in our package documentation.

Reply:
I added https://github.com/open-telemetry/opentelemetry.io/pull/3275/commits/b162a9a4b923033ef8880bdba3f6228368e297ae and created https://github.com/open-telemetry/opentelemetry-go/issues/4526

Reply:
I want to do it as a separate PR and first address the comment in OTel Go: https://github.com/open-telemetry/opentelemetry-go/issues/4526

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/4527